include/variant_detection/bam_functions.hpp

```diff
@@ -5,6 +5,26 @@
 #include <seqan3/io/sam_file/output.hpp>
 #include <seqan3/utility/char_operations/predicate.hpp>
 
+
+/*!\details
+ * From the SAM/BAM Format Specification
+ * | Bit  | Bit   | Description                                                       | Note                           |
+ * |:----:|:-----:|:-----------------------------------------------------------------:|:------------------------------:|
+ * |    1 | 0x1   | template having multiple segments in sequencing                   |                                |
+ * |    2 | 0x2   | each segment properly aligned according to the aligner            | necessary for read pair method |
+ * |    4 | 0x4   | segment unmapped                                                  | will be skipped                |
+ * |    8 | 0x8   | next segment in the template unmapped                             |                                |
+ * |   16 | 0x10  | SEQbeing reverse complemented                                     | is checked in split read method |
```

Comment:
Is there a blank space missing in this and in the next line ?
SEQ beging ..
SEQ of ...